<launch>
    <rosparam command="load" file="$(find skyscanner)/launch/conf/fg.yaml" />
    <group ns="mesonh">
        <rosparam command="load" file="$(find skyscanner)/launch/conf/mesonh.yaml" />
    </group>

    <param name="use_sim_time" value="true" />
    <param name="clock_time_scale" value="1" />

    <node pkg="skyscanner"
          type="clock_generator.py"
          name="clock_generator"
          required="true"/>

    <node pkg="skyscanner"
          type="atmosphere.py"
          name="atmosphere"
          required="true"/>

    <node pkg="skyscanner"
          type="flightgearenvironment.py"
          name="flightgearenvironment"/>

    <node pkg="skyscanner"
          type="flightgearuav.py"
          name="flightgearuav"/>
</launch>
